lib/srv/proxy.go

```diff
@@ -247,7 +247,10 @@ func (t *proxySubsys) proxyToHost(
 	// if port is 0, it means the client wants us to figure out
 	// which port to use
 	useExactPort := len(t.port) > 0 && t.port != "0"
-	ips, _ := net.LookupHost(t.host)
+	log.Debugf("proxy connecting to host=%v port=%v, exact port=%v\n", t.host, t.port, useExactPort)
+
+	ips, err := net.LookupHost(t.host)
+	log.Debugf("host lookup (%v)=%v, err=%v", t.host, ips, err)
```

Comment:
what do you mean by "grep catches it all the time"? adding error logging here helped me solve DNS-related issue on a customer POC.